oshmem/runtime/oshmem_shmem_finalize.c

```diff
@@ -58,30 +58,30 @@
 #include "oshmem/shmem/shmem_lock.h"
 #include "oshmem/runtime/oshmem_shmem_preconnect.h"
 
+extern int oshmem_shmem_globalexit_status;
+
 static int _shmem_finalize(void);
 
 int oshmem_shmem_finalize(void)
 {
     int ret = OSHMEM_SUCCESS;
-    static int32_t finalize_has_already_started = 0;
     int32_t _tmp = 0;
```

Comment:
This value is no longer used.

Reply:
Thanks! Will re-push soon.

Reply:
@sam6258 I think I already deleted it in this commit, it is a '-' at beginning :)

Reply:
@xinzhao3 I was referring to _tmp